config/jobs/kubernetes/sig-testing/dependencies.yaml

```diff
@@ -33,8 +30,8 @@ presubmits:
   - name: pull-kubernetes-dependencies-canary
     path_alias: "k8s.io/kubernetes"
     decorate: true
-    branches:
-    - master
```

Comment:
will this one have any side effects? (we also have feature branches)

Reply:
Yes it will have a side-effect of correctly ensuring that feature branches pass this job, which IMO we want.